source/_posts/2024-08-21-hacs-the-best-way-to-share-community-made-projects.markdown

```diff
@@ -0,0 +1,113 @@
+
+This is one of the biggest updates yet for HACS, but if you're not sure what HACS is or why you might consider using it, check out our [explainer below](#what-is-hacs).
+
+### New frontend <!-- omit from toc -->
+
+<img src='/images/blog/2024-08-hacs2/frontend.png' style='border: 0;box-shadow: none;' alt="HACS frontend looks like data tables">
+
+HACS 2.0 [main dashboard](https://www.hacs.dev/docs/use/repositories/dashboard/) has taken cues from Home Assistant, and now closely matches the native look and functionality of the data tables you find on the entities or automation pages. This includes the options to filter, group, sort, and search.
+
+### Faster downloads <!-- omit from toc -->
+
+<img src='/images/blog/2024-08-hacs2/downloads.png' style='border: 0;box-shadow: none;' alt="HDownload window going very fast">
+
+Previously, HACS 100% relied on GitHub to retrieve information, from file locations to the number of stars, so we needed to limit the API (as there are a lot of HACS users). To speed things up, we've created a [remote dataset](https://www.hacs.dev/docs/faq/data_sources/) stored in Cloudflare R2 buckets, which are updated at regular intervals.  Files are still downloaded from GitHub and their API is still contacted, but it will see drastically fewer calls, and the speed improvement is massive. Behind the scenes, this was a big piece of work, which has ongoing costs, and shows the kind of support the Open Home Foundation can provide to a community-driven project like HACS.
+
+### Update and repair <!-- omit from toc -->
+
+<img src='/images/blog/2024-08-hacs2/updates-repairs.png' style='border: 0;box-shadow: none;' alt="update all your Home Assistant from the same place">
+
+No more visiting the HACS page every day to check for updates. They'll now appear in the same places system and add-on [updates](https://my.home-assistant.io/redirect/updates/) appear in the native Home Assistant format. Also, once they're finished updating, they will give you suggested repairs (for instance, reloading the dashboards or rebooting Home Assistant).
+
+### Other improvements <!-- omit from toc -->
+
+We have also renamed things to help them make more sense, including changing "category" to "type", and "Lovelace" to "dashboard" (Lovelace needs to make room for [Grace](https://www.home-assistant.io/blog/2024/03/04/dashboard-chapter-1/#what-is-project-grace)). We're also including Template management, which utilizes the new [template type](https://www.hacs.dev/docs/publish/template/) to enhance your Jinja templates.
+
+{% note %}
+**Breaking changes** - There have been some breaking changes, such as removing the yaml configuration, no longer including the NetDaemon type, and moving [beta selection to a switch entity](https://www.hacs.dev/docs/use/entities/update/). Many of the changes are more likely to affect those sharing their code via HACS, check the [release notes for the full list](https://github.com/hacs/integration/releases/tag/2.0.0).
```

Comment:
```suggestion
**Breaking changes** - There have been some breaking changes, such as removing the YAML configuration, no longer including the NetDaemon type, and moving [beta selection to a switch entity](https://www.hacs.dev/docs/use/entities/update/). Many of the changes are more likely to affect those sharing their code via HACS, check the [release notes for the full list](https://github.com/hacs/integration/releases/tag/2.0.0).
```

Reply:
Thank you for the catch! :fishing_pole_and_fish:
https://github.com/home-assistant/home-assistant.io/pull/34400